src/specBuilder/area/areaSpecBuilder.test.ts

```diff
@@ -45,6 +45,7 @@ const defaultAreaProps: AreaSpecProps = {
 	color: DEFAULT_COLOR,
 	dimension: DEFAULT_TIME_DIMENSION,
 	index: 0,
+	markType: 'area',
```

Comment:
I added a property `markType` to bar, area, line, scatter and donut. This is helpful for chart tooltip since it could be added to any of these. With the markType prop, chart tooltip can be setup correctly for the correct mark type.